spec/lib/tasks/eu_exit_business_finder_spec.rb

```diff
@@ -0,0 +1,172 @@
+require 'rake'
+require 'rails_helper'
+require 'gds_api/test_helpers/publishing_api_v2'
+
+RSpec.describe 'eu_exit_business_finder:update_subscriber_list_titles_and_facet_values' do
+  include GdsApi::TestHelpers::PublishingApiV2
+
+  before :each do
+    Rails.application.load_tasks
+  end
+
+  def mock_facet_values_to_remove(values)
+    allow(EuExitFacetMigrationConfig).to receive(:facet_values_to_remove).and_return(values)
+  end
+
+  def mock_facet_values_to_replace(replacements)
+    allow(EuExitFacetMigrationConfig).to receive(:facet_values_to_replace).and_return(replacements)
+  end
+
+  def mock_facet_value_labels_to_replace(replacements)
+    allow(EuExitFacetMigrationConfig).to receive(:facet_value_labels_to_replace).and_return(replacements)
+  end
+
+  def stub_find_eu_exit_guidance_business_json
+    abs_path_to_fixture_data = Rails.root.join('spec/lib/tasks/fixtures/find-eu-exit-guidance-business.json')
```

Comment:
For info `Rails.root.join` is pretty cool it that it lets you get around operating system defaults by letting you only specify the directory names. So here, you could have used:

```
Rails.root.join('spec', 'lib', 'tasks', 'fixtures', 'find-eu-exit-guidance-business.json')
```

Reply:
Ah, so that it comes out as `spec\lib\tasks\...etc` on Windows? Not as readable but I see the benefit 👍 

We seem to use it lots of different ways in email-alert-api (and doubtless elsewhere), probably worth changing it to a consistent approach in a separate PR someday.